Xamarin.PropertyEditing.Mac/PropertyTableDelegate.cs

```diff
@@ -130,9 +130,13 @@ public override void ItemDidCollapse (NSNotification notification)
 		public override nfloat GetRowHeight (NSOutlineView outlineView, NSObject item)
 		{
 			var facade = (NSObjectFacade)item;
-			var vm = facade.Target as PropertyViewModel;
 			var group = facade.Target as IGroupingList<string, PropertyViewModel>;
-			string cellIdentifier = (group == null) ? vm.Property.Name : group.Key;
+			if (group != null) {
+				return 30;
+			}
+
+			var vm = facade.Target as PropertyViewModel;
+			string cellIdentifier = (group == null) ? vm.GetType ().Name : group.Key;
 
 			var editor = (PropertyEditorControl)outlineView.MakeView (cellIdentifier + "edits", this);
```

Comment:
Does this need to be unique? I notice in the other part of the code you removed the `cellIdentifier` prefix

Reply:
The other removes the prefix because it is purely a label, the view type is always the same. This is only measuring property editors which are unique to VM types. Really line 139 could be simplified as it never should be a group key as that's already returned on 135. I'll clean that up.